src/ui/citation-popup/view.js

```diff
@@ -129,3 +145,23 @@ function main () {
     }
 
 }
+
+function renderRef (item, data) {
+    const ref = data.value;
+    const author = he.encode(ref["author"]);
+    const year = he.encode(ref["year"]);
+    const title = he.encode(ref["title"]);
```

Comment:
Those are encoded variables so their names should reflect that. Have you read the Making Wrong Code Look Wrong article? https://www.joelonsoftware.com/2005/05/11/making-wrong-code-look-wrong/ In this article he mentions prefixing the variables with "s" (as "safe"). I tend to add an "Html" suffix. Either way, please pick a convention for encoded variable names.

Alternatively, you can simply encode the variable at the point where you insert it into HTML and that way you don't need an intermediate variable.

Reply:
Done